#automat map

test_device:

  address: 172.21.176.51
  port: 502
  
  publish_rate: 10 #Hz 
  refresh_rate: 100 #Hz 
  state_rate: 1 #Hz 
    
  connected_IO:
    digital_input: 16
    digital_output: 8
    analog_input: 8
    analog_output: 8
    
  offsets:
    digital_input: 0
    digital_output: 0
    analog_input: 0
    analog_output: 2048
      
  publish_on_timer: 
    - di_in_test1
    - di_in_test2
    - di_out_test1
    - an_out_test1
    - an_out_test2
  
  publish_on_event:
    - di_in_test1

  input:
    digital:
      di_in_test1: 1
      di_in_test2: 2
      di_in_test5: 5
    analog:
      
      
  output:
    digital:
      di_out_test1: 1
      di_out_test2: 2
    analog:
      an_out_test1: 2
      an_out_test2: 4
